--- a/lib/lpc176x/device/system_LPC17xx.c
+++ b/lib/lpc176x/device/system_LPC17xx.c
@@ -514,7 +514,7 @@ uint32_t SystemCoreClock = __CORE_CLK;/*!< System Clock Frequency
 (Core Clock)*/
  * @brief  Setup the microcontroller system.
  *         Initialize the System.
  */
-void SystemInit (void)
+void __attribute__((externally_visible)) SystemInit (void)
 {
 #if (CLOCK_SETUP)                       /* Clock Setup                        */
   LPC_SC->SCS       = SCS_Val;
